src/config.rs

```diff
@@ -194,13 +194,14 @@ impl Context {
 // use the provided config path or `$DRGCFG` value if set
 // otherwise will default to $XDG_CONFIG_HOME
 // fall back to `$HOME/.config` if XDG var is not set.
-// todo crossplatform support
 fn eval_config_path(path: Option<&str>) -> String {
     match path {
         Some(p) => p.to_string(),
         None => env::var("DRGCFG").unwrap_or_else(|_| {
-            let xdg = env::var("XDG_CONFIG_HOME")
-                .unwrap_or(format!("{}/.config", env::var("HOME").unwrap()));
+            let xdg = match config_dir() {
+                Some(path) => path.into_os_string().into_string().unwrap(),
+                None => String::from("."),
```

Comment:
Please specify a default path

Reply:
I would error out if a path cannot be found. The error message should suggest the user to use the `--config` option

Reply:
Added !